### ISSUE TITLE
macros `@for`, `@if` etc.

### ISSUE THREAD

**Issue by hhaensel** [GenieFramework/Stipple.jl#233]
Today I found out, that it is indeed possible to define macros `@if`, `@for` etc.

So shouldn't we rename our strange macros which seem very unintuitive and difficult to remember for me.

```julia
julia> macro var"for"(expr)
         Expr(:kw, Symbol("v-for"), esc_expr(expr))
       end

julia> list(@recur("fruit in fruits"), "{{ fruit }}")
"<q-list v-for=\"fruit in fruits\">{{ fruit }}</q-list>"
```

**Comment by essenciary** [comment]
Yes! We came up with `@recur` as I didn't know how to define conflicting-named macros with `var"..."`.

**Comment by hhaensel** [comment]
I'll do it then and keep the old ones as deprecated

**Comment by hhaensel** [comment]
Just had to learn that only from julia 1.10 on this syntax is supported :-(

**Comment by hhaensel** [comment]
Solved by #234
